packages/tasks/src/model-libraries.ts

```diff
@@ -263,6 +263,22 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: { term: { path: "model.safetensors" } },
 	},
+	multi_token_prediction: {
+		prettyLabel: "Multi-Token Prediction",
+		repoName: "multi-token-prediction",
+		repoUrl: "https://huggingface.co/facebook/multi-token-prediction",
```

Comment:
```suggestion
		repoUrl: "https://huggingface.co/facebook/multi-token-prediction",
```

Is this expected to be the only repo from that library? Is there a GitHub repo with the code? (asking because I can't see the files in this HF repo yet)

Reply:
No it looks like the HF repository contains all necessary files to run inference, there's no Github repository